pages/resources.js

```diff
@@ -1,15 +1,160 @@
+import Link from "next/link";
+import { useState } from "react";
 import styled from "styled-components";
 
 const Resources = () => {
   return (
     <div>
       <Headline>Hilfestellen und Beratungsangebote</Headline>
       <List>
```

Comment:
i would recommend to create a separate datasheet for your list items. Something like:

const data = [
{name: "Hilfetelefon Gewalt gegen Frauen",
link: "https://www.hilfetelefon.de/"
tel: "08000 116 016"
},
....
...
]

If you do so, maintaining the data is much easier. To render the data, you could easily use the array.map() function. Code becomes much cleaner afterwards as well.